changelog/549.bugfix.rst

```diff
@@ -0,0 +1 @@
+Make InteractionData\* dataclasses dicts (:class:`InteractionData`, :class:`ApplicationCommandInteractionData`, and so on).
```

Comment:
I don't see what that star symbol is for?

Reply:
Ah, its in the wrong place.

Its supposed to mean all classes that end with 
InteractionData.

Reply:
https://github.com/DisnakeDev/disnake/pull/600/commits/657236de8d25c6b3540fa247cd8917f84f369fec